requirements/base.txt

```diff
@@ -18,6 +18,7 @@ jsonfield==1.0.2
 openpyxl==2.2.6
 python-magic==0.4.15
 subprocess32==3.5.4
+Wand==0.5.5
```

Comment:
Odkial viem ci [Wand](http://docs.wand-py.org/en/0.5.5/) potrebuje este nejake externe kniznice ? Pre instalaciu vyzadovalo
 `$ apt-get install libmagickwand-dev`
`$ pip install Wand`

Na konvertovanie obrzka do PDF sa vyskytla chyba `wand.exceptions.PolicyError: not authorized`. Chyba sa da odstranit upravou suboru  `/etc/ImageMagick-6/policy.xml` prepisanim atributu `rights` z `none` na `read|write`. Ako to zahrnut do setup-u / nastavenia?

Reply:
> Odkial viem ci Wand potrebuje este nejake externe kniznice?

```
$ env/bin/pip freeze > before
$ env/bin/pip install Wand
$ env/bin/pip freeze > after
$ diff before after
```

Alebo priamo v zdrojakoch: https://github.com/emcconville/wand/blob/master/setup.py#L40

Oba sposoby hovoria, ze nema ziadne dependencies.

Reply:
> ` $ apt-get install libmagickwand-dev`

Pls pridaj do README informaciu, ze treba `libmagickwand-dev` podobne ako som tam pridal libreoffice v https://github.com/martinmacko47/chcemvediet/commit/81b371f8919f1676e823cb8749714f4035f60b47

> Na konvertovanie obrzka do PDF sa vyskytla chyba wand.exceptions.PolicyError: not authorized. Chyba sa da odstranit upravou suboru /etc/ImageMagick-6/policy.xml prepisanim atributu rights z none na read|write. Ako to zahrnut do setup-u / nastavenia?

Kedze to meni systemove nastavenia, tak `setup.py` to spravit nemoze. Napis o tom odstavec do README pod odstavec, kde budes pisat, ze `libmagickwand-dev` je dependency. Na obe miesta aj pre local dev aj pre production postup.

Reply:
:ok: